### PR TITLE
feat(ironfish): Add `wallet/getAccountTransactionDescriptions`

### PR DESCRIPTION
**Pull request by rohanjadvani**
## Summary

Add an RPC endpoint that gets the descriptions from an unsigned transaction

## Testing Plan

Unit test

## Documentation

Does this change require any updates to the Iron Fish Docs (ex. [the RPC API
Reference](https://ironfish.network/docs/onboarding/rpc/chain))? If yes, link a
related documentation pull request for the website.

```
[x] Yes
```

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish/src/rpc/routes/wallet/getAccountTransactionDescriptions.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    sender: string
+    value: string
+  }[]
+  sentNotes: {
+    assetId: string
+    memo: string
+    owner: string
+    sender: string
+    value: string
+  }[]
```
Comment (patnir) [inline ironfish/src/rpc/routes/wallet/getAccountTransactionDescriptions.ts:40]: would it be possible to reuse one or more of our existing RPC objects like this https://ironfish.network/developers/documentation/rpc/objects/rpcWalletTransaction

File: ironfish/src/rpc/routes/wallet/getAccountTransactionDescriptions.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+export const GetAccountTransactionDescriptionsRequestSchema: yup.ObjectSchema<GetAccountTransactionDescriptionsRequest> =
```
Comment (jowparks) [inline ironfish/src/rpc/routes/wallet/getAccountTransactionDescriptions.ts:43]: This isn't getting the descriptions, this is decrypting the unsigned transaction. Technically I think it should be able to take in a signed or unsigned transaction as well.

Comment (jowparks) [reply ironfish/src/rpc/routes/wallet/getAccountTransactionDescriptions.ts:43]: Aren't descriptions supposed to be the publically viewable contents of a transaction? For mints/burns they just happen to be public in general. The notes are private

Comment (rohanjadvani) [reply ironfish/src/rpc/routes/wallet/getAccountTransactionDescriptions.ts:43]: It's a mix of both I guess. It's returning the mint/burn descriptions but also decrypting the notes associated with the output descriptions on the unsigned transaction.

The primary use case here is just to be for the unsigned transaction to show more details. But if you think it's useful to expand this, I can rework it or rename it so it's more explicit for unsigned transactions.

**Pull request closed without merging**